### PR TITLE
Feat/optional pin

### PR DESCRIPTION
**Pull request by shamb0t**
Pass optional arguments to `log.append`

### CODE REVIEW

File: src/Store.js [hunk 1]
```diff
@@ -470,14 +469,14 @@ class Store {
     }
   }
 
-  async _addOperation (data, batchOperation, lastOperation, onProgressCallback) {
+  async _addOperation (data, { batchOperation, lastOperation, onProgressCallback, pin = false } = {}) {
```
Comment (haadcode) [inline src/Store.js]: This means the API changes and needs changes in all stores. How about instead adding `pin` as the last arg and keep the current API?

Comment (shamb0t) [reply src/Store.js]: @haadcode Seems to me all the stores need to be changed regardless in order to pass the options, no?

Comment (haadcode) [reply src/Store.js]: To pass (use/enable) the option yes, but if they don't change anything, their old code still works as expected, whereas changing the API would require all dependants to change their `_addOperation()` calls to match the changed API.

**Comment by haadcode** [comment]
LGTM 👍

**Pull request merged**